wweval/R/supplement_analyses_and_figs.R

```diff
@@ -665,15 +665,63 @@ get_stats_improved_forecasts <- function(scores,
       "location", "forecast_date"
     )) |>
     dplyr::mutate(
-      pct_change_crps = (ww - hosp) / hosp
+      pct_change_crps = (ww - hosp) / hosp,
+      rel_crps = ww / hosp
+    )
+
+  relative_crps_raw <- scores |>
+    compute_relative_crps(id_cols = c(
+      "location", "forecast_date", "date"
+    )) |>
+    dplyr::mutate(
+      pct_change_crps = (ww - hosp) / hosp,
+      rel_crps = ww / hosp
     )
 
+  ggplot(relative_crps_by_forecast) +
+    geom_histogram(aes(x = pct_change_crps))
+
+  ggplot(relative_crps_raw) +
+    geom_histogram(aes(x = pct_change_crps))
+  ggplot(relative_crps_raw) +
+    geom_histogram(aes(x = rel_crps)) +
+    scale_x_continuous(trans = "log10")
+
+
+
+
+  forecasts_way_worse <- relative_crps_by_forecast |>
+    dplyr::filter(pct_change_crps > 3)
```

Comment:
Why is this done by percent change rather than simply by relative CRPS?

Reply:
I think they are interchangeable, can edit.

Reply:
I'd prefer to keep everything in terms of simple fold-change (i.e. `rel_crps `) rather than fractional fold change (`rel_crps - 1`).

Reply:
Ok that is fine and I think makes sense -- though just to be clear in the paper we are reporting like so: "The wastewater-informed model performed 3% worse than the hospital admissions only model (rscore = 1.03 [0.103/0.100])".

Reply:
Ok just would like your check to make sure I am interpreting the symmetry correctly -- see the change to the threshold part.

Reply:
This is now addressed.